### DISCUSSION TITLE
Formating for my projects tab to show more projects per row

### DISCUSSION THREAD

**Discussion by TheQwasr** [ScratchAddons/ScratchAddons#7674] [Ideas]
As the title says it would be nice to have an extension that formats the "my projects" tab to show multiple projects per row. This will be useful if you have many projects to not have to scroll down really far. The number of projects per row should be able to be configured.

**Comment by Samq64** [comment]
There's already an addon called "More items per row" but it doesn't have an option my stuff.

**Reply by TheQwasr** [reply]
Then i think an option for my project would be useful
